server/apps/aap/import_text_archive/commands.py

```diff
@@ -152,7 +153,7 @@ def _process_bunch(self, x):
             item['firstcreated'] = utc.normalize(value) if value.tzinfo else value
             item['versioncreated'] = item['firstcreated']
 
-            item['unique_id'] = doc.find('dcdossier').get('unique')
+            generate_unique_id_and_name(item)
             item['ingest_id'] = id
```

Comment:
Sorry, should have written more clearly. :) Is the _value_ of this attribute meant to be a function?